vhost-device-scmi/src/devices/common.rs

```diff
@@ -323,7 +322,7 @@ pub trait SensorT: Send {
         } else {
             self.format_unit(0)
         };
-        let name = self.sensor().name.clone();
+        let name = self.sensor().name.clone().unwrap();
```

Comment:
can the `name` be `None` and panic here?

Reply:
Yes, it will panic here. If there is no 1, and 2 which I [mentioned](https://github.com/rust-vmm/vhost-device/pull/710#discussion_r1747849968).
We will use default name to promise that it is not **None**.

Reply:
okay, so maybe a comment here would be nice.

Reply:
In branch from [1fb2f52 ](https://github.com/rust-vmm/vhost-device/commit/1fb2f52e500f9fe6821e31a54fdf1f8c231c9b72)to [533416e](https://github.com/rust-vmm/vhost-device/commit/533416e21d45c7cc4b82072e6a146ac9bb82456f),
I leave a comment here.